packages/core/src/docs/data-model/text-x/__tests__/utils.spec.ts

```diff
@@ -339,4 +341,39 @@ describe('test text-x utils', () => {
             }],
         });
     });
+
+    it('test isUselessRetainAction()', () => {
```

Comment:
Should describe expected results in `it`.

Reply:
I think expect expresion already explain the expect result, it also has different expected in this case.